packages/box/test/box.js

```diff
@@ -4,7 +4,7 @@ const assert = require("assert");
 const inquirer = require("inquirer");
 const sinon = require("sinon");
 const Config = require("@truffle/config");
-const Box = require("../");
+const { default: Box, normalizeSourcePath  }= require("../");
 const TRUFFLE_BOX_DEFAULT =
   "https://github.com:trufflesuite/truffle-init-default";
```

Comment:
I wonder if we want to eventually move to using the bare-box in the truffle-box repo instead of this one...

Reply:
I think it is probably mainly historically significant. Actually we don't even use this any more. `truffle init` copies files and no longer requires an internet connection. Maybe we can update some tests to use that infrastructure instead of unbox.